CONFIGURATION.md

```diff
@@ -9,8 +9,13 @@ A sample configuration can be found at [config-sample.json](https://github.com/a
 | log_level  | Y        | String | Broker Log Level (DEBUG, INFO, ERROR, FATAL)
 | username   | Y        | String | Broker Auth Username
 | password   | Y        | String | Broker Auth Password
+| master_password_seed | Y        | String | Seed to generate DB instances master passwords
+| broker_name | Y        | String | Unique name to identify by tag in RDS the instances created by this broker
```

Comment:
I have updated `broker_name` property description in spec in https://github.com/alphagov/paas-aws-broker-boshrelease/pull/8 to "Unique name of RDS broker, used to construct a tag for instance identification". I think we should use same string here...